include/Track.h

```diff
@@ -245,6 +271,11 @@ class TrackContentObjectView : public selectableObject, public ModelView
 	void setBBPatternBackground( const QColor & c );
 	void setGradient( const bool & b );
 	void setMouseHotspotHand(const QSize & s);
+	
+	QColor color() const
+	{
+		return m_tco->color();
+	};
```

Comment:
It looks like this function isn't used - remove it?

Reply:
I got a build error when I removed it, with a function in Track.cpp is using it, so keep it around?

Reply:
If removing it causes an error, then yes, keep it around. I don't get an error removing it on my machine though - what error do you get?

Reply:
Ah, I'm sorry, it builds now because the function that used it was removed xD. Removed the function.